examples/vision/adamatch.py

```diff
@@ -69,7 +69,7 @@
 from tensorflow import keras
 from tensorflow.keras import layers
 from tensorflow.keras import regularizers
-from official.vision.image_classification.augment import RandAugment
+from official.vision.ops.augment import RandAugment
```

Comment:
What version of the model garden are you using? Make sure the version is listed in the example.